lib/db/db_test.go

```diff
@@ -305,7 +305,7 @@ func TestRepairSequence(t *testing.T) {
 		{Name: "duplicate", Blocks: genBlocks(2)},
 		{Name: "missing", Blocks: genBlocks(3)},
 		{Name: "overwriting", Blocks: genBlocks(4)},
-		{Name: "inconsistent", Blocks: genBlocks(5)},
+		{Name: "inconsistent", Blocks: genBlocks(2)},
```

Comment:
Comment on why it's important to have this specific number of blocks? (I guess to get the same blocks as another file?)